app/models/incomeSourceDetails/viewmodels/ViewIncomeSourcesViewModel.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.incomeSourceDetails.viewmodels
+
+import java.time.LocalDate
+
+case class ViewIncomeSourcesViewModel(viewSoleTraderBusinesses: List[ViewBusinessDetailsViewModel],
+                                      viewUkProperty: Option[ViewPropertyDetailsViewModel],
+                                      viewForeignProperty: Option[ViewPropertyDetailsViewModel],
+                                      viewCeasedBusinesses: List[ViewCeasedBusinessDetailsViewModel])
+
+case class ViewBusinessDetailsViewModel(tradingName: String, tradingStartDate: Option[LocalDate])
```

Comment:
from what I can see in API definition / story: tradingName and tradingStartDate - can be Optional; you might need to change these as well ;